src/data/notion-data.ts

```diff
@@ -40,6 +40,25 @@ export const peopleBySlug: Record<string, Person> = {
 		role: "Open Source",
 		slug: "aakansha-doshi",
 	},
+	"aaron-harper": {
+		bio: "I’m a software developer in Ann Arbor, Michigan. I’m a fan of writing reliable code, my dog Benny, and attending my favorite local meetup: Michigan TypeScript.  When I’m not helping developers implement durable workflows, I love spending time with my wife and daughter, gardening, and building orchestration engines that combine event driven architecture and workflows.",
```

Comment:
```suggestion
		bio: "I’m a software developer in Ann Arbor, Michigan. I’m a fan of writing reliable code, my dog Benny, and attending my favorite local meetup: Michigan TypeScript. When I’m not helping developers implement durable workflows, I love spending time with my wife and daughter, gardening, and building orchestration engines that combine event driven architecture and workflows.",
```

Reply:
lol.  fixed in f9f5e938a2905d86dea37642a0c2916b8b374c98